.github/workflows/convert_notebooks.yml

```diff
@@ -6,6 +6,7 @@ on:
   push:
     branches:
     - 'main'
+    - 'ea/docs_fix' # TODO remove after test
```

Comment:
Should it be removed now or after the merge?

Reply:
I remove it before merge, I just want to make sure that conversion build passed, when it succeed, I remove